cogs/induct.py

```diff
@@ -238,6 +237,21 @@ async def _perform_induction(self, ctx: TeXBotApplicationContext, induction_memb
                 reason=f"{ctx.user} used TeX Bot slash-command: \"/induct\"",
             )
 
+        intro_channel: discord.TextChannel | None = discord.utils.get(
+            self.bot.main_guild.text_channels,
+            name="introductions",
+        )
+
+        tex_emoji: discord.Emoji | None = self.bot.get_emoji(743218410409820213)
+
+        if intro_channel:
+            recent_message: discord.Message
+            for recent_message in await intro_channel.history(limit=30).flatten():
+                if recent_message.author.id == induction_member.id:
+                    if tex_emoji: await recent_message.add_reaction(tex_emoji)  # noqa: E701
```

Comment:
`if` statements should use expanded version:

```python
if tex_emoji:
    await recent_message.add_reaction(tex_emoji)
```

Reply:
PEP8 can suck my ass <3

Reply:
PEP8 my beloved, don't listen to him, I love you.